assignments/array-methods.js

```diff
@@ -58,11 +58,20 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 let fullName = [];
 console.log(fullName);
 
+runners.forEach((getNames)=>{
+    let names = getNames.first_name + " " + getNames.last_name;
+    fullName.push(names)
+  });
+
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
 console.log(allCaps); 
 
+const caps = runners.map((capitalize));{
+    caps.push(capitalize.first_name.toUpperCase())
```

Comment:
You don't need to push to the array caps because it's the array that is being returned by the map. You can instead write:
```suggestion
   const caps = runners.map((capitalize));{
     return capitalize.first_name.toUpperCase()
   }
```
Whatever you put as the return statement for the map will be added to the new array that map creates, which is caps in this case.